moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -197,53 +197,79 @@ bool Servo::validateParams(const servo::Params& servo_params) const
   bool params_valid = true;
   auto robot_state = planning_scene_monitor_->getStateMonitor()->getCurrentState();
   auto joint_model_group = robot_state->getJointModelGroup(servo_params.move_group_name);
+  std::string check_yaml_msg = " Check the parameters YAML file used to launch this node.";
```

Comment:
Make this const and consider renaming it into something other than msg to avoid confusion with a ROS msg
```suggestion
  const std::string check_yaml_string = " Check the parameters YAML file used to launch this node.";
```